change/@ni-nimble-tokens-b264b0ee-0471-4955-99c5-10e8b68dff74.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Note that these changes will cause a Chromatic diff because we have a story that displays all the base tokens. But the current diffs are incorrect as they show some colors from a previous version of this PR. This is likely due to a known issue with how Chromatic builds are triggered. To retrigger the snapshots you can modify the guid at the bottom of `packages/storybook/.storybook/preview.js` and include that change in your PR.

Reply:
Modified GUID and the storybook build got updated